File: week-2/week2-4.py
def twoSum(nums, target):
    num=len(nums)
    for i in range(num):
        a1=target-nums[i]
        if a1>0:
            for j in range(i+1,num):
                if nums[j] == a1:
                    ans=[]
                    ans.extend([i,j])
                    return ans

result=twoSum([2, 11, 7, 15], 9)
print(result)

File: week-2/week2-4.js
function twoSum(nums, target){
    let num=nums.length
    for(let i=0; i<num; i++){
        let a1=target-nums[i]
        if(a1>0){
            for(let j=i+1; j<num; j++){
                if(nums[j]==a1){
                    let ans=[]
                    ans.push(i,j)
                    return ans
                }
            }
        }
    }
}
let result=twoSum([2, 11, 7, 15], 9);
console.log(result);